src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -857,13 +850,13 @@ private void RefreshConsolidatablePackagesCount()
                 NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
                     await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                    _topPanel._labelConsolidate.Count = 0;
+                    _topPanel.ShowCountOnConsolidateTab(count: 0);
                     var loadContext = new PackageLoadContext(ActiveSources, Model.IsSolution, Model.Context);
                     var packageFeed = await CreatePackageFeedAsync(loadContext, ItemFilter.Consolidate, _uiLogger);
                     var loader = new PackageItemLoader(
                         loadContext, packageFeed, includePrerelease: IncludePrerelease);
 
-                    _topPanel._labelConsolidate.Count = await loader.GetTotalCountAsync(100, CancellationToken.None);
+                    _topPanel.ShowCountOnConsolidateTab(count: await loader.GetTotalCountAsync(100, CancellationToken.None));
```

Comment:
The first named parameter is not necessary, but a named parameter is necessary for the magical `100` argument.